.github/scripts/get_tutorials_stats.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
+import os.path
+from typing import List, Optional, Tuple, Dict, Union
+import boto3  # type: ignore[import]
+from botocore.exceptions import ClientError  # type: ignore[import]
+from pprint import pprint
+
+dynamodb = boto3.resource('dynamodb', region_name="us-east-1")
+
+def run_command(cmd: str, cwd: Optional[str] = None) -> str:
+    """
+    Run a shell command.
+
+    Args:
+        cmd: Command to run
+        cwd: Working directory
+    Returns:
+        Output of the command.
+    """
+    import shlex
+    from subprocess import check_output
+
+    return check_output(shlex.split(cmd), cwd=cwd).decode("utf-8")
+
+
+def get_history(cwd: Optional[str] = None) -> List[List[str]]:
+    """
+    Get commit history from git.
+    Args:
+        cwd: Working directory
+    Returns:
+        List of commit hashes
+    """
+    lines = run_command(
+        "git log --date=short --pretty=format:" '%h;"%an";%ad;"%s"' " --shortstat",
+        cwd=cwd,
+    ).split("\n")
+
+    def parse_string(line: str) -> str:
+        """
+        Parse a line into a list of strings.
```

Comment:
I'm confused by this description. This is clearly returning a single string, not a list.

Reply:
Updated